examples/python/evm/scoped_account.custom_node.py

```diff
@@ -0,0 +1,55 @@
+# Usage: uv run python evm/scoped_account.custom_node.py
```

Comment:
Example looks good, but is technically not using a custom node, so let's name the file `scoped_account.send_transaction.py`